### ISSUE TITLE
User can't connect due to bad packet compression

### ISSUE THREAD

**Issue by SulpherStaer** [RelativityMC/RaknetFabric#3]
```
23.02 01:25:11 [Server] Netty Epoll Server IO #2/INFO [STDOUT]: Krypton detected, applying compatibility
23.02 01:25:11 [Server] User Authenticator #28/INFO UUID of player [USERNAME REDACTED] is [UUID REDACTED]
23.02 01:25:11 [Server] Netty Epoll Server IO #2/INFO [STDERR]: io.netty.handler.codec.DecoderException: Badly compressed packet - size of 236 is below server threshold of 512
23.02 01:25:11 [Server] Netty Epoll Server IO #2/INFO [STDERR]: io.netty.handler.codec.DecoderException: Badly compressed packet - size of 120 is below server threshold of 512
23.02 01:25:11 [Disconnect] User com.mojang.authlib.GameProfile@REDACTED[id=REDACTED,name=REDACTED,properties={textures=[com.mojang.authlib.properties.Property@REDACTED]},legacy=false] (/IP REDACTED) has disconnected, reason: Internal Exception: io.netty.handler.codec.DecoderException: Badly compressed packet - size of 236 is below server threshold of 512
```

The Network Compression Threshold on the server is actually set to 128. I do not know what version of the mod the player has, I have told them to update to 1.4 - don't know yet if this has fixed the issue.

They were not using Raknet when they had this issue.

**Comment by KJP12** [comment]
Seems related to #1, except with an alternative reproduction step of manually setting the compression threshold lower in `server.properties`.

The work around seems to be setting the compression threshold to match the amount the error shows, `512`, as #1 cannot be reproduced when the threshold is set to `512`.

**Comment by ishland** [comment]
Fixed in 1e46f5c
